src/Appwrite/Utopia/Response/Model/AttributeEmail.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Appwrite\Utopia\Response\Model;
+
+use Appwrite\Utopia\Response;
+use Appwrite\Utopia\Response\Model\AttributeString;
+
+class AttributeEmail extends AttributeString
+{
+    public function __construct()
+    {
+        $this
+            ->addRule('format', [
```

Comment:
We need to normalize the structure and abstract the complexity of extending custom attributes. I don't think this should be an exact representation of the DB lib or the internal data structure.

I would use `type = email` instead of `type=string` and drop the format attribute.

Reply:
Now that the code is more complete, would you still like to make this change? Now that `format` is a string, it fits a lot better with `type`, highlighting how we are sticking to primitives in the DB.